open-hackathon-client/src/client/views/__init__.py

```diff
@@ -139,7 +139,7 @@ def __get_api(url, headers=None, **kwargs):
 @app.context_processor
 def utility_processor():
     def get_now():
-        return __date_serializer(datetime.now())
+        return __date_serializer(datetime.utcnow())
```

Comment:
OK. However the name of function "get_now" has to be changed to avoid name conflicts.